src/data/auth/projectServices.ts

```diff
@@ -0,0 +1,37 @@
+import {
```

Comment:
Project's services aren't part of the auth folder. Please, create `projects` folder inside `data` folder and move this file there